doc/sphinx/proof-engine/ltac.rst

```diff
@@ -407,34 +413,38 @@ We can branch with the following structure:
 
    Branching is left-associative.
 
-First tactic to work
-~~~~~~~~~~~~~~~~~~~~
+First tactic to succeed
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Backtracking branching may be too expensive. In this case we may
-restrict to a local, left biased, branching and consider the first
-tactic to work (i.e. which does not fail) among a panel of tactics:
+Backtracking branching may be too expensive. We can restrict
+backtracking to local, left biased branching and use the first
+tactic that succeeds from a list of tactics:
 
-.. tacn:: first [{*| @ltac_expr}]
+.. tacn:: first [ {*| @ltac_expr } ]
    :name: first
 
    The :n:`@ltac_expr__i` are evaluated to :n:`v__i` and :n:`v__i` must be
    tactic values for i = 1, ..., n. Supposing n > 1,
-   :n:`first [@ltac_expr__1 | ... | @ltac_expr__n]` applies :n:`v__1` in each
+   :tacn:`first` :n:`[@ltac_expr__1 | ... | @ltac_expr__n]` applies :n:`v__1` in each
    focused goal independently and stops if it succeeds; otherwise it
    tries to apply :n:`v__2` and so on. It fails when there is no
    applicable tactic. In other words,
-   :n:`first [@ltac_expr__1 | ... | @ltac_expr__n]` behaves, in each goal, as the first
+   :tacn:`first` :n:`[@ltac_expr__1 | ... | @ltac_expr__n]` behaves, in each goal, as the first
    :n:`v__i` to have *at least* one success.
 
+   :tacn:`first` is an :token:`ltac_expr1`.
+
    .. exn:: No applicable tactic.
       :undocumented:
 
+   .. todo PR I couldn't find an ltac definition for the following.  Is it still supported?
```

Comment:
It was introduced in #676. I stopped investigating at this point, but it should be possible to determine if this still exists or not.

Reply:
For the record, it is still supported indeed, as shown by the example displayed below. And contrary to what the description seemed to suggest, it is not defined as an Ltac definition in a `.v` file.

Reply:
It doesn't behave like normal tactics:

```
Ltac Y := first idtac.  (* accepted *)
first [idtac].  (* accepted *)
first idtac.  (* Expected a list *)
Y.  (* Expected a list *)
```

We can come back to this later.

Reply:
Yep, I don't really understand this.